src/components/PathStep/index.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars, react/no-unused-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function PathStep(props) {
+    return <></>;
+}
+
+PathStep.propTypes = {
+    /** The name of the PathStep. */
+    name: PropTypes.string,
```

Comment:
here name does not match with typescript where it also accepts a number, I think we should accept numbers too